src/rp2_common/pico_platform/include/pico/platform.h

```diff
@@ -423,6 +425,15 @@ static inline void busy_wait_at_least_cycles(uint32_t minimum_cycles) {
     );
 }
 
+/*! \brief Get the current core number
+ *  \ingroup hardware_sync
```

Comment:
Does this ingroup need updating?

Reply:
yes; and i fixed that yesterday (online) - bad github